ontologies/WIP/sierratransformable

```diff
@@ -327,7 +327,7 @@
    /* Populate wwork:subjects people and organisations. Rules:
    * 
    * 1. For all bibliographic records use "marcTag": "600", "610". 
-   * 2. Populate platform "label" for "600" and "610" concatenated with subfields "a", "c", "d" "e"  type as "Subject". 
+   * 2. Populate platform "label" for "600" and "610" concatenated with subfields "c", "a", "b", "d", "e"  type as "Subject".
```

Comment:
Is the corresponding transformer change not done yet, or have I just missed it?

Reply:
The transformer change for 610 will come in another P, this is only for 600. ALso the rules for 610 are probably going to be different from what the sierratransformable file says (check #2556 ). But we need input from @silveroliver about those